app/src/components/RequestTable.js

```diff
@@ -22,65 +26,172 @@ function RequestTable({ requests, token, onSubmit, onWithdraw }) {
     [onWithdraw]
   )
 
+  const getEntries = useMemo(() => {
+    console.log('ownRequests ', ownRequests)
+    return !ownRequests
+      ? requests
+          .sort(({ date: dateLeft }, { date: dateRight }) =>
```

Comment:
Why not do the sort  in the reducer? Otherwise, each time the tab is changed the requests are re-sorted.